server/requirements.txt

```diff
@@ -15,4 +15,6 @@ timm==0.6.13
 typeguard==2.13.3
 accelerate==0.18.0
 pytesseract==0.3.10
-gradio==3.24.1
\ No newline at end of file
+gradio==3.24.1
+setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
+wheel>=0.38.0 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
**Suggestion**: Ensure the updated version of wheel (0.38.0) is compatible with the project's build and deployment processes. It might be necessary to adjust CI/CD scripts if they rely on specific behavior of the wheel package. <br> **Code Suggestion**: 
 ```
 +wheel>=0.38.0 # not directly required, pinned by Snyk to avoid a vulnerability
 ```